users/tests.py

```diff
@@ -1,27 +1,22 @@
+import faker
 from django.test import TestCase
+
 from users.models import User
 
-import faker
 
 class UserTestCase(TestCase):
-  def setUp(self):
-      self.fake=faker.Faker()
-      self.first_name=self.fake.first_name()
-      self.last_name=self.fake.last_name()
-      self.email=self.fake.email()
-      self.gender='NS'
-
-      User.objects.create(
-          first_name=self.first_name,
-          last_name=self.last_name,
-          email=self.email,
-          gender=self.gender
-      )
+    def setUp(self):
+        self.fake = faker.Faker()
+        self.first_name = self.fake.first_name()
+        self.last_name = self.fake.last_name()
+        self.email = self.fake.email()
+        self.gender = 'NS'
 
-  def test_user_attributes(self):
-      user = User.objects.get(email=self.email)
-      self.assertEqual(user.first_name, self.first_name)
-      self.assertEqual(user.last_name, self.last_name)
-      self.assertEqual(user.email, self.email)
-      self.assertEqual(user.gender, self.gender)
+        User.objects.create(first_name=self.first_name, last_name=self.last_name, email=self.email, gender=self.gender)
```

Comment:
Just a comment, here you can "force" black into using the hanging indent by adding a trailing comma after `self.gender`. That way it will be formatted like this:
```suggestion
        User.objects.create(
            first_name=self.first_name,
            last_name=self.last_name,
            email=self.email,
            gender=self.gender,
        )
```
No need to change it, just letting you know in case you prefer that way